<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketOperation</class>
 <widget class="QWidget" name="TicketOperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>票务操作 Ticket Operation</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>起始站/From</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>日期/Date</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>终点站/To</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_date">
      <property name="text">
       <string>（格式：01-31）</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_from"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_to"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>160</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="do_1query">
      <property name="text">
       <string>查票/Query Ticket</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="do_2transfer">
      <property name="text">
       <string>查询换乘/Query Transfer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="do_3buy">
      <property name="text">
       <string>购票/Buy Ticket</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>248</width>
     <height>197</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>上侧日历供您参考。</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>The calendar is for your reference.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
